install/unix/apache2.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: e443388d4249fea383d6ec2d1a338be220cf95ee Maintainer: fabiosal Status: ready -->
```

Comment:
The revision of this file is not correct. [Here you can find the list of all the revisions for this file](https://github.com/php/doc-en/commits/master/install/unix/apache2.xml).